internal/app/cli/list_test.go

```diff
@@ -88,14 +89,44 @@ func TestList(t *testing.T) {
 		r.On("List", mock.Anything, sp).Return(listRes, nil).Once()
 
 		// when: list from the repo
-		err := List(context.Background(), repoSpec, sp)
+		err := List(context.Background(), repoSpec, sp, OutputFormatPlain)
 		stdout := getStdout()
 		stderr := getStderr()
 
 		// then: there is no error
 		assert.NoError(t, err)
 		// and then: stdout outputs the listable ThingModel
 		assert.Contains(t, stdout, listRes.Entries[0].Name)
+		assert.NotContains(t, stdout, "{")
+		// and then: stderr has no outputs
+		assert.Equal(t, "", stderr)
+	})
+	t.Run("with ok output json", func(t *testing.T) {
+		restoreStdout, getStdout := testutils.ReplaceStdout()
+		restoreStderr, getStderr := testutils.ReplaceStderr()
+		defer restoreStdout()
+		defer restoreStderr()
+
+		// given: a RepoManager and a repo having 1 ThingModel
+		repoSpec := model.NewRepoSpec("r1")
+		r := mocks.NewRepo(t)
+		rMocks.MockReposGet(t, rMocks.CreateMockGetFunction(t, repoSpec, r, nil))
+
+		var sp *model.SearchParams
+
+		r.On("Spec").Return(repoSpec).Maybe()
+		r.On("List", mock.Anything, sp).Return(listRes, nil).Once()
+
+		// when: list from the repo
+		err := List(context.Background(), repoSpec, sp, OutputFormatJSON)
+		stdout := getStdout()
+		stderr := getStderr()
+
+		// then: there is no error
+		assert.NoError(t, err)
+		// and then: stdout outputs the listable ThingModel
+		assert.Contains(t, stdout, fmt.Sprintf("\"name\": \"%s\"", listRes.Entries[0].Name))
+		assert.Contains(t, stdout, "{")
```

Comment:
We should check the json output here as you did it in the other tests...
```
var actual any
err = json.Unmarshal([]byte(stdout), &actual)
assert.NoError(t, err)
```

Reply:
fixed